bl31/aarch64/bl31_entrypoint.S

```diff
@@ -42,6 +43,39 @@
 	 */
 
 func bl31_entrypoint
+#ifdef BL31_ENABLE_RELOCATION
```

Comment:
Optional: Consider passing BL31_DIFFERENT_LOAD_RUN_ADDRESS through the build system by using the add_define make function in the top level Makefile. If you do this, BL31_DIFFERENT_LOAD_RUN_ADDRESS should be documented in the user guide.